js/usage-stats-chart.js

```diff
@@ -89,6 +89,9 @@
 					color: 'rgba(0,0,0,0.05)',
 					drawTicks: false,
 				},
+				ticks: {
+					beginAtZero: true
```

Comment:
Probably something that we need to add in the https://github.com/pkp/ui-library/blob/main/src/components/Chart/LineChart.vue as well? Not sure.

Reply:
Yes please, you can merge it after applying the fix. Thanks @bozana !